### DISCUSSION TITLE
home sparito

### DISCUSSION THREAD

**Discussion by carlone1** [indomus/forum#93] [HUB: Home Assistant]
salve a tutti
stavo modificando il file di configurazione e alla fine ho riavviato
a quel punto non è piu ripartito, ho spento riacceso il raspberry  ho inserito l'indirizzo  homeassistan.tlocal:8123

QUESTA PAGINA NON FUNZIONA IN QUESTO MOMENTO
homeassistant local - non ha inviato dati
err_empty_response

ho guardato sul router myfastgate e nei dispositivi c'è ed è online

Ho guardato sulla scheda SD nel raspberry e il programma esiste ancora

Per caso è gia successo a qualcun'altro? Avete qualche consiglio?  grazie

**Comment by carlone1** [comment]
aggiornamento
sono riuscito ad entrare ed ho capito che avendo aggiunto DOCK DNS dovevo entrare con **https**
cmq mi fa entrare con la scritta rossa di sito non sicuro
vorrei mandarvi un immagine ma credo che qui non sia possibile, la metterò nei messaggi sulla pag. dockdns

ho messo un immagine su indomus "installare dockdns su Hassio" nei messaggi

**Reply by snakuzzo** [reply]
DUCKDNS...non DOCKDNS

La tua domanda è stata fatta innumerevoli volte sul sito, sul vecchio forum ed in chat su Telegram. Ed è stata fornita altrettante volte la stessa risposta.
Tu segui la guida...abiiliti l'https...ma a mio avviso dovresti sapere un minimo di cosa si sta parlando. Inoltre ti inviterei a non postare le cose sia qui che lì...deciditi! Inoltre, se proprio non puoi fare  ameno di postare immagini, è meglio che tu lo faccia qui che nei commenti sul blog.

Ad ogni quello che posso dirti è che quel warning "è del tutto normale".
Il certificato che stai utilizzando è stato rilasciato dalla CA Let's Encrypt su base FQDN, quindi se ti colleghi utilizzando l'indirizzo IP al posto del tuo FQDN di duckdns fallisce l'handshake ed è del tutto normale che ti dia quel warning.

**Reply by carlone1** [reply]
scusa ho messo l'immagine di la perchè non conosco questo forum e non vedo come si possa postare l'immagine qui.
Si ho sbagliato a scrivere duck.

comunque credo che il mio FQDN sia questo, **https://xxxxxxxx.duckdns.org:8123/** oppure **https://xxxxxxxxx.duckdns.org/**
se è questo non mi funziona ancora.
per cortesia confermami che è uno di questi 2, perchè se è cosi devo andare a cercare cosa ho sbagliato nella configurazione
grazie.

**Reply by snakuzzo** [reply]
`xxxxxx.duckdns.org` è il tuo FQDN 
`https` è il protocollo
`8123` è la porta

Continuo a ribadire che devi chiarirti dei concetti base, altrimenti è inutile che vai avanti

**Reply by carlone1** [reply]
Scusa ma non capisco  perchè è inutile che vada avanti, sono solo 2 settimane che che sono in questo mondo, ho installato cose che non avevo mai visto prima, ho configurato un sacco di add on, posso anche magari sbagliare qualcosa non credi?.
Poi quali concetti base devo chiarire?

**Reply by snakuzzo** [reply]
Il mio consiglio (ripeto) è che dovresti approfondire alcuni concetti basilari prima di andare avanti. Tutto qua.
Te lo dico perché dalle domande e dalla difficoltà che stai avendo ora ho capito che sei in difficoltà con alcuni concetti.
Quello che ti ho suggerito è ... "andare a vedere cos'è un certificato, cos'è l'https, cos'è un FQDN, etc"

**Reply by snakuzzo** [reply]
Tornando al tuo problema...se hai il warning ad accedere tramite il tuo FQDN, vuol dire o che il certificato è sbagliato o che la tua configurazione è sbagliata.
Controlla il log di HA quando provi ad effettuare l'accesso...di sicuro scriverà qualcosa nel file di log
